docs/index.html

```diff
@@ -38,11 +38,11 @@
     </style>
 </head>
 <body>
-    <section class="hero is-medium is-warning is-bold">
+    <section class="hero is-medium is-light is-bold">
```

Comment:
ここはaddしてはいけない箇所ですので、元に戻してください。
```suggestion
    <section class="hero is-medium is-warning is-bold">
```